hnn_core/network.py

```diff
@@ -219,25 +228,52 @@ def __repr__(self):
               % (self.n_of_type['L2_basket'], self.n_of_type['L5_basket']))
         return '<%s | %s>' % (class_name, s)
 
-    def _create_src_list(self):
-        """ creates the immutable source list along with corresponding numbers of cells
+    def _assign_external_feeds_coords(self, n_common_feeds, p_unique_keys):
```

Comment:
didn't we discuss to remove this method and move the contents into `_add_external_feeds`? The less class methods we have the better.

Reply:
OK, I've flattened more now that I have a better "view" on things. All this will be replaced, though.